### PR TITLE
chore(connectiond): enable sentry for connectiond

### PR DESCRIPTION
**Pull request by alexzurbonsen**
## Summary

Enable Sentry monitoring for ConnectionD.

## Test Plan

Build locally with Bazel and make. Tested connection to Sentry with a private Sentry account and provoking a crash in ConnectionD's main.

## Additional Information

Done in pairing with @ajahl.

### CODE REVIEW

File: lte/gateway/c/connection_tracker/src/main.cpp [hunk 1]
```diff
@@ -65,14 +66,17 @@ static uint32_t get_log_verbosity(const YAML::Node& config,
 }
 
 int main(void) {
-  magma::init_logging(CONNECTION_SERVICE);
 
   auto mconfig = load_mconfig();
   auto config =
       magma::ServiceConfigLoader{}.load_service_config(CONNECTION_SERVICE);
+  magma::init_logging(CONNECTION_SERVICE);
```
Comment (sebathomas) [inline lte/gateway/c/connection_tracker/src/main.cpp]: Why move this here?

**Review by themarwhal — APPROVED** [review]
We may need to also update Ci as we did with sctpd cc @sebathomas

File: lte/gateway/c/connection_tracker/src/main.cpp (hunk 1 above)
Comment (alexzurbonsen) [reply lte/gateway/c/connection_tracker/src/main.cpp]: We mirrored the respective part in the sctpd main here, but there is no necessity for that.

File: lte/gateway/c/connection_tracker/src/main.cpp [hunk 2]
```diff
@@ -65,14 +66,17 @@ static uint32_t get_log_verbosity(const YAML::Node& config,
...
   magma::set_verbosity(get_log_verbosity(config, mconfig));
   MLOG(MINFO) << "Starting Connection Tracker";
 
+  sentry_config_t sentry_config = construct_sentry_config_from_mconfig();
+  initialize_sentry(SENTRY_TAG_CONNECTIOND, &sentry_config);
```
Comment (themarwhal) [inline lte/gateway/c/connection_tracker/src/main.cpp]: we probably want to start sentry as early as possible. what are we blocked on above?

File: lte/gateway/c/connection_tracker/src/BUILD.bazel [hunk 3]
```diff
@@ -22,6 +22,7 @@ cc_binary(
         "//lte/protos:mconfigs_cpp_proto",
         "//orc8r/gateway/c/common/config:mconfig_loader",
         "//orc8r/gateway/c/common/service303",
+        "//orc8r/gateway/c/common/sentry:sentry_wrapper",
```
Comment (Neudrino) [inline lte/gateway/c/connection_tracker/src/BUILD.bazel:24]: Previous lexicographical ordering should be retained.

File: lte/gateway/c/connection_tracker/src/main.cpp (hunk 2 above)
Comment (alexzurbonsen) [reply lte/gateway/c/connection_tracker/src/main.cpp]: Sounds like the more suitable choice. Changed the order accordingly.

File: lte/gateway/c/connection_tracker/src/main.cpp [hunk 4]
```diff
@@ -73,6 +76,7 @@ int main(void) {
   magma::set_verbosity(get_log_verbosity(config, mconfig));
   MLOG(MINFO) << "Starting Connection Tracker";
 
+
```
Comment (Neudrino) [inline lte/gateway/c/connection_tracker/src/main.cpp]: Is this change needed?

**Comment by Neudrino** [comment]
There was no shutdown here, opposed to https://github.com/magma/magma/pull/11459/files#diff-6d090aca871507fc00b60cb35cf1ae06203826f1e2170eabe88ee27843723978R118. Don't we need that?

**Comment by alexzurbonsen** [comment]
> There was no shutdown here, opposed to https://github.com/magma/magma/pull/11459/files#diff-6d090aca871507fc00b60cb35cf1ae06203826f1e2170eabe88ee27843723978R118. Don't we need that?

Ah, I wasn't aware of this. Thank you. Added it as well.

**Review by Neudrino — APPROVED** [review]

**Review by ajahl — APPROVED** [review]
lgtm

**Review by themarwhal — APPROVED** [review]

**Review by quentinDERORY — APPROVED** [review]

**Review by sebathomas — APPROVED** [review]

**Review by tmdzk — APPROVED** [review]
